### PR TITLE
Added section in readme to ask anyone for help in improving the docs

### LINKED ISSUE #1331: Clarify Chinese documentation
We just merged #1321 which adds translations for almost all of our documentation to Chinese, amazing! We should clarify in the English and Chinese readme's that if anyone is available to help improve the docs, to please send a pull request. Actually, wouldn't be a bad idea to add a note to our English docs that help improving them is always welcome :smile:

### PR DESCRIPTION
**Pull request by Zaccc123**
Fixes #1331

Hi, let me know if anything needs to improve.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -292,6 +292,14 @@ Please note that this project is released with a Contributor Code of Conduct. By
...
 If you add or remove a source file from Moya, a corresponding change needs to be made to the `Moya.xcodeproj` project at the root of this repository. This project is used for Carthage. Don't worry, you'll get an automated warning when submitting a pull request if you forget.
 
+### Help us improve Moya documentation
+Whether you’re a core member or a user trying it out for the first time, you can make a valuable contribution to Moya by improving the documentation. Help us by:
+
+- sending us feedback about something you thought was confusing or simply missing
+- suggest for better wording or ways of explaining certain topics
+- sending us a pull request via GitHub
+
+
```
Comment (sunshinejr) [inline Readme.md]: Could we also add here that we'd love some help on Chinese documentation as well? 😉

**Comment by sunshinejr** [comment]
Hey @Zaccc123, thank you very much for this PR! I left you one comment, nothing big. Let me know if it makes sense to you!

File: Readme.md [hunk 2]
```diff
@@ -292,6 +292,14 @@ Please note that this project is released with a Contributor Code of Conduct. By
 
 If you add or remove a source file from Moya, a corresponding change needs to be made to the `Moya.xcodeproj` project at the root of this repository. This project is used for Carthage. Don't worry, you'll get an automated warning when submitting a pull request if you forget.
 
+### Help us improve Moya documentation
+Whether you’re a core member or a user trying it out for the first time, you can make a valuable contribution to Moya by improving the documentation. Help us by:
+
+- sending us feedback about something you thought was confusing or simply missing
+- suggest for better wording or ways of explaining certain topics
```
Comment (SD10) [inline Readme.md]: suggest -> suggesting better
This matches the rest of the verb forms used in the other two points

Comment (Zaccc123) [reply Readme.md]: sounds good, I have updated it.

File: Readme.md (hunk 1 above)
Comment (Zaccc123) [reply Readme.md]: done! 😄

**Comment by ashfurrow** [comment]
Thanks a lot for contributing to Moya! I've invited you to join the Moya GitHub organization – no pressure to accept! If you'd like more information on what that means, check out our [contributor guidelines](https://github.com/Moya/contributors) and feel free to reach out with any questions.

**Pull request merged**